### ISSUE TITLE
Improve main README.md. Add link to the programs 😽

### ISSUE THREAD

**Issue by mpotane** [mpotane/PythonScriptPH#13]
### Example:
- [Algorithms](https://github.com/mpotane/PythonScriptPH/tree/main/Algorithms)

**Comment by Vishvam10** [comment]
I can do it. Please assign the task to me.

**Comment by mpotane** [comment]
> I can do it. Please assign the task to me.

Nice bro i'll do that in a sec.

**Comment by Lakshayainani** [comment]
@mpotane Can I give it a try ?
I'm new to github contributions.

PR: https://github.com/mpotane/PythonScriptPH/pull/14

**Comment by Lakshayainani** [comment]
https://github.com/mpotane/PythonScriptPH/pull/14

**Comment by Lakshayainani** [comment]
@mpotane Anymore changes required for this issue?

**Comment by mpotane** [comment]
> @mpotane Can I give it a try ? I'm new to github contributions.
> 
> PR: #14

Let's wait  @Vishvam10 a lil bit more

**Comment by Lakshayainani** [comment]
@mpotane Any suggestions on how to get started and contribute more?

**Comment by mpotane** [comment]
> @mpotane Any suggestions on how to get started and contribute more?

check #5 out

**Comment by Vishvam10** [comment]
I'm almost done with it. I just wrote a small script to get all the folders and its link. Do you want me to commit the script as well ? If so, I would like to improve it. It's pretty basic at the moment.

**Comment by mpotane** [comment]
> I'm almost done with it. I just wrote a small script to get all the folders and its link. Do you want me to commit the script as well ? If so, I would like to improve it. It's pretty basic at the moment.

you can add new  script, I will check and review the code. But your open PR needs to be merged first.

**Comment by Vishvam10** [comment]
Cool. I'll submit the updated README now. I'm new to this, should I open a new PR or an issue for the script ?

**Comment by mpotane** [comment]
> Cool. I'll submit the updated README now. I'm new to this, should I open a new PR or an issue for the script ?

Okay don't push changes yet to main just open a new issue.

**Comment by mpotane** [comment]
> Cool. I'll submit the updated README now. I'm new to this, should I open a new PR or an issue for the script ?

yep you can submit your PR now. If yours turn out to be okay, i'll mark the other one duplicate.

**Comment by Vishvam10** [comment]
I submitted the PR for the README file. I have a few doubts regarding the scripts. Say the folder structure was something like this :

- Parent_Folder_1
    - Sub_Folder_1
    - Sub_Folder_2
    - ...
- Parent_Folder_2
- ...

Should the README contain nested links in the following fashion :

- [ Parent_Folder_1 ]( Link )
    - [ Sub_Folder_1 ]( Link )
    - [ Sub_Folder_2 ]( Link )
    - ...
- [ Parent_Folder_2 ]( Link )
- ...

Or should it just have the link for the Parent_Folders ?

**Comment by mpotane** [comment]
> I submitted the PR for the README file. I have a few doubts regarding the scripts. Say the folder structure was something like this :
> 
> * Parent_Folder_1
>   
>   * Sub_Folder_1
>   * Sub_Folder_2
>   * ...
> * Parent_Folder_2
> * ...
> 
> Should the README contain nested links in the following fashion :
> 
> * [ Parent_Folder_1 ](Link)
>   
>   * [ Sub_Folder_1 ](Link)
>   * [ Sub_Folder_2 ](Link)
>   * ...
> * [ Parent_Folder_2 ](Link)
> * ...
> 
> Or should it just have the link for the Parent_Folders ?

We can open a new issue so more can PR in hacktoberfest

**Comment by Vishvam10** [comment]
Cool. I'll probably include that as a flag. I'll develop a CLI to dynamically update the readme with new links whenever a new folder is found. Should I proceed with this ? And if I can, should I create a new folder named "Scripts" to have all these files ? 

One last thing, I've registered to hacktoberfest as a code contributor. So the readme pr won't count right ? And if the CLI part looks good and I develop it, would that be counted ? 

I'm new to this and any help / guidance would be greatly appreciated !
